x/bank/types/codec.go

```diff
@@ -29,6 +29,10 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 		&SendAuthorization{},
 	)
 
+	registry.RegisterImplementations(
+		&UpdateDenomMetadataProposal{},
+	)
```

Comment:
The first argument here should be the interface type. In this case:
```suggestion
	registry.RegisterImplementations(
		(*govtypes.Content)(nil),
		&UpdateDenomMetadataProposal{},
	)
```